README.md

```diff
@@ -44,6 +44,7 @@ pip install -e "python[all]"
 ```
 
 ### Notes
+- FlashInfer is turn on by default on accelerating inference using highly optimized CUDA kernels. Installation see [install FlashInfer](https://docs.flashinfer.ai/installation.html). More information see [flashinfer.md](docs/flashinfer.md).
```

Comment:
Duplicate this under `Method 1` and `Method 2`. Make it easier to read. Remove the file `docs/flashinfer.md`